packages/p2p/src/stores/sendbird-store.ts

```diff
@@ -325,10 +329,6 @@ export default class SendbirdStore extends BaseStore {
                         // eslint-disable-next-line no-console
                         console.warn(error);
                     });
-            } else {
-                (async () => {
-                    await this.markMessagesAsRead(true);
-                })();
```

Comment:
dont need to handle reading the messages here as we already do it through registerEventListeners function